pupa/cli/commands/update.py

```diff
@@ -59,6 +59,46 @@ def save_report(report, jurisdiction):
             )
 
 
+@transaction.atomic
+def save_report(report, jurisdiction):
+    from pupa.models import RunPlan
+    plan = RunPlan.objects.filter(jurisdiction=jurisdiction).latest('id')
+
+    # from pupa.reports.organizations import organization_report
+    from pupa.reports.people import person_report
+    from pupa.reports.posts import post_report
+    from pupa.reports.bills import bill_report
+    from pupa.reports.votes import vote_report
+
+    scrapers = report['plan']['scrapers']
+
+    measures = {}
+
+    # organization_report(jurisdiction)
+
+    if 'people' in scrapers:
```

Comment:
Hurm, this depends on the scrapers to be named in a certen way (and people can be added in non-person scrapes) -- can we just always run all the hooks?